### PR TITLE
Day two lab

### PR DESCRIPTION
**Pull request by tcoots88**
- [x] Top-level README “Table of Contents” is updated
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [X] Expected failure
     - [X] Edge Case (if applicable/obvious)
 - [ ] Feature tasks for this challenge are completed
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [X] Link to code

### CODE REVIEW

File: src/main/java/com/tcoots/songr/HomeController.java [hunk 1]
```diff
@@ -20,6 +24,28 @@ public String getAlbums(Model m) {
         m.addAttribute("albumArray", albums);
         return "albums";
     }
+//Should direct to page showing all albums
+    @GetMapping("allAlbums")
+    public String getAllAlbums(Model m) {
+        Iterable<Album> albums = albumRepository.findAll();
```
Comment (JamesDansie) [inline src/main/java/com/tcoots/songr/HomeController.java]:
```suggestion
        List<Album> albums = albumRepository.findAll();
```

File: src/main/java/com/tcoots/songr/AlbumRepository.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.tcoots.songr;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface AlbumRepository extends JpaRepository<Album, Long> {
+    public List<Album> findByTitle(String albumName);
```
Comment (JamesDansie) [inline src/main/java/com/tcoots/songr/AlbumRepository.java:8]: Your ablums don't have titles, they have albumNames;
`    public List<Album> findByalbumName(String albumName);
`
